### ISSUE TITLE
Allow passing candidates to be based on binding OR percentile cutoff.

### ISSUE THREAD

**Issue by malachig** [griffithlab/pVACtools#1142]
Based the current docs it sounds a candidate it filtered out unless it passes both of these cutoffs.  

```
  -b BINDING_THRESHOLD, --binding-threshold BINDING_THRESHOLD
                        Report only epitopes where the mutant allele has ic50
                        binding scores below this value. (default: 500)
  --percentile-threshold PERCENTILE_THRESHOLD
                        Report only epitopes where the mutant allele has a
                        percentile rank below this value. (default: None)
```

Is that a correct interpretation for what would happen if you set a value for both of these?

It would be nice if this could be interpreted as an either OR parameter.  So that for example, if a candidate is <500 nm OR <1% it would be considered a pass.

**Comment by susannasiebert** [comment]
Yes, that is correct. Is that a general change to the behavior we want to make or something that should be configurable?

**Comment by malachig** [comment]
It just keeps adding more complexity, but I guess it could be nice to have this configurable based on intent:

- One wishes to be conservative, so one requires BOTH criteria to be met to get a pass
- One wishes to get more candidates, so one required EITHER criteria to be met to get a pass
